doc/release-notes/release-6.sgml

```diff
@@ -57,17 +209,68 @@ This section gives an account of those changes in three categories:
 <sect1>New directives<label id="newdirectives">
 <p>
 <descrip>
-	<p>There have been no directives added.
+	<tag>paranoid_hit_validation</tag>
+	<p>Controls whether to perform extra internal checks when loading
+	   entries from the on-disk cache
+
+	<tag>cache_log_message</tag>
+	<p>Configure logging options on a per-message basis, overriding the
+	   per-section options. Message IDs are guaranteed stable across builds and
+	   releases. Only a few messages support this for now.
 </descrip>
 
 <sect1>Changes to existing directives<label id="modifieddirectives">
 <p>
 <descrip>
-	<p>There have been no directives changed.
+	<tag>time units</tag>
+	<p>All directives accepting time values now accept a time unit suffix
+	   from nanosecond to decade
+
+	<tag>sslcrtvalidator_program</tag>
+	<p>Add cache and ttl specifiers to limit resources allocated
```

Comment:
These are optional, thus "options".

The "cache" option has not actually changed. Just more verbose text about its use.

The "ttl=infinity" setting is new to the existing "ttl=" option. This text should just mention what it means.

Also, literal syntax should be wrapped in EM tags for highlighting.
```suggestion
	<p>New <em>ttl=infinity</em> option to disable TTL expiry on stored helper responses.
```